engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoSource.java

```diff
@@ -1,44 +1,72 @@
 package org.enso.interpreter.runtime.data;
 
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.library.CachedLibrary;
+import com.oracle.truffle.api.library.ExportLibrary;
+import com.oracle.truffle.api.library.ExportMessage;
 import com.oracle.truffle.api.source.Source;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
 
 /** Wrapper for exposing sources to Enso. Delegates to original methods with no behavior changes. */
-public class EnsoSource {
+@ExportLibrary(InteropLibrary.class)
+public final class EnsoSource implements EnsoObject {
+  private static final String[] MEMBERS = {
+    "getLanguage", //
```

Comment:
nitpick but what are these trailing `//` meant to be?

Reply:
It helps the formatting. I'd like each string to be on its own line. The `//` is a way to convince the _non-configurable, opinionated google formatter_ to honor human friendly formatting rather than force its own illogical formatting rules on us.

Reply:
Oh, nice 😅